lib/stripe/errors.rb

```diff
@@ -25,6 +25,7 @@ def initialize(message = nil, http_status: nil, http_body: nil,
       @http_status = http_status
       @http_body = http_body
       @http_headers = http_headers || {}
+      @idempotent_replayed = @http_headers["idempotent-replayed"] == "true"
```

Comment:
We're confident that the value returned by Stripe will always be lowercase, right? No need to call `.downcase` to normalize?

Reply:
Yeah, good question — I double-checked the implementation and am 100% sure.

I thought about just doing something like a `.downcase` anyway, but decided against since it doesn't seem too likely that we'd ever change that fact, and having the `.downcase` there would somewhat imply that we might expect the value to sometimes be in other cases.